test/cases_matching_callbacks/2.btx_callbacks.yaml

```diff
@@ -0,0 +1,9 @@
+- :name: usr
+  :regex: \S*_1
+  :args:
+  - :name: _timestamp
+    :type: int64_t
+  - :name: cf_1
+    :type: const char *
+  - :name: pf_1
+    :type: uint64_t
```

Comment:
Why we need the type? It should already be in the model no?

Reply:
You are right, we don't actually need the type. I did the simple approach, but I can obtain the attributes of the events that match. This is what we need?